src/Playwright.TestingHarnessTest/tests/nunit.spec.ts

```diff
@@ -335,3 +335,81 @@ test.describe('Expect() timeout', () => {
     expect(result.stdout).toContain("LocatorAssertions.ToHaveTextAsync with timeout 123ms")
   });
 });
+
+test.describe('Parallelizable analyzer', () => {
```

Comment:
consider testing the analyzer using pure C# and verifying the analyzer creates a diagnostic - https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix#build-unit-tests

Reply:
Yeah, I thought about that. But as we already have all the test harness set up and the overall goal is pretty much the same, I went that way.
I would have loved the Test harness suite to be a C# project :)